src/SonarScanner.MSBuild.PreProcessor/WebServer/SonarQubeWebServer.cs

```diff
@@ -113,19 +113,19 @@ public override async Task<IList<SensorCacheEntry>> DownloadCache(ProcessedArgs
                 ? await base.DownloadComponentProperties(component)
                 : await DownloadComponentPropertiesLegacy(component);
 
-        protected override Uri AddOrganization(Uri uri) =>
+        protected override string AddOrganization(string uri) =>
             serverVersion.CompareTo(new Version(6, 3)) < 0 ? uri : base.AddOrganization(uri);
 
         private async Task<IDictionary<string, string>> DownloadComponentPropertiesLegacy(string projectId)
         {
             var uri = GetUri("api/properties?resource={0}", projectId);
-            logger.LogDebug(Resources.MSG_FetchingProjectProperties, projectId, uri);
+            logger.LogDebug(Resources.MSG_FetchingProjectProperties, projectId);
             var result = await ExecuteWithLogs(async () =>
             {
                 var contents = await downloader.Download(uri, true);
                 var properties = JArray.Parse(contents);
                 return properties.ToDictionary(p => p["key"].ToString(), p => p["value"].ToString());
-            }, uri);
+            });
```

Comment:
You removed the URL from the parameters but not from the error log string. Not sure what value it brings now that log.

Reply:
I agree this does not bring much value now; I would be up to remove the `ExecuteWithLogs` method. WDYT?